Makefile

```diff
@@ -21,6 +25,13 @@ build-cross:
 	$(call go-build,linux,s390x)
 	$(call go-build,windows,amd64)
 	$(call go-build,windows,386)
+	$(call go-build-noselinux,linux,amd64)
+	$(call go-build-noselinux,linux,arm)
+	$(call go-build-noselinux,linux,arm64)
+	$(call go-build-noselinux,linux,ppc64le)
+	$(call go-build-noselinux,linux,s390x)
+	$(call go-build-noselinux,windows,amd64)
+	$(call go-build-noselinux,windows,386)
```

Comment:
since we're using gnu make extensions here already, maybe it makes sense to put these arches into a separate variable and use `$(foreach ...)` to avoid repetition (at least for linux)?

Reply:
e.g.
```make
linux-cross-arches := adm64 386 ....
$(foreach a, $(linux-cross-arches), $(eval $(call go-build, linux, $(a))))
```